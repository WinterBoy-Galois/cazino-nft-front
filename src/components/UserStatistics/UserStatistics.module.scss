@import '../../styles/shared';

.value {
  justify-content: flex-end;

  @include media-breakpoint-up(md) {
    justify-content: flex-start;
  }
}

.username {
  &__hidden {
    color: $live-updates-light-blue;
  }
}
