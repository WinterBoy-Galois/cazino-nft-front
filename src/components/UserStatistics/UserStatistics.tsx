import React from 'react';
import styles from './UserStatistics.module.scss';
import DetailList from '../DetailList';
import BitcoinValue from '../BitcoinValue';
import { formatBitcoin } from '../../common/util/format.util';
import BitcoinProfit from '../BitcoinProfit';
import GameIconAndText from '../GameIconAndText';
import LostBets from '../LostBets';
import { useTranslation } from 'react-i18next';
import { UserStatistic } from '../../models/userStatistics.model';

interface IProps {
  userStatistic: UserStatistic;
}

const UserStatistics: React.FC<IProps> = ({
  userStatistic: { totalWager, luckyBets, mostPlayed, totalBets, totalProfit },
}) => {
  const { t } = useTranslation();

  return (
    <DetailList
      details={[
        {
          label: t('userStatistics.totalWager'),
          value:
            totalWager !== null ? (
              <BitcoinValue value={formatBitcoin(totalWager)} className={styles.value} />
            ) : (
              <div className={styles.username__hidden}>{t('hidden')}</div>
            ),
        },
        {
          label: t('userStatistics.totalProfit'),
          value:
            totalProfit !== null ? (
              <BitcoinProfit value={totalProfit} className={styles.value} />
            ) : (
              <div className={styles.username__hidden}>{t('hidden')}</div>
            ),
        },
        {
          label: t('userStatistics.mostPlayed'),
          value: <GameIconAndText game={mostPlayed} />,
        },
        {
          label: t('userStatistics.totalBets'),
          value: totalBets,
        },
        {
          label: t('userStatistics.wonBets'),
          value: luckyBets,
        },
        {
          label: t('userStatistics.lostBets'),
          value: <LostBets totalBets={totalBets} luckyBets={luckyBets} />,
        },
      ]}
    />
  );
};

export default UserStatistics;
