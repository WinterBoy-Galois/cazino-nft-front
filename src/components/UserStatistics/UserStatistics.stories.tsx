import React from 'react';

import UserStatistics from '.';
import { GameTypes } from '../../models/gameTypes.model';

export default {
  title: 'Components/UserStatistics',
  component: UserStatistics,
};

const data = {
  userInfo: {
    totalWager: 0,
    totalProfit: 0,
    mostPlayed: GameTypes.CLAMS,
    totalBets: 27,
    luckyBets: 4,
    hideUsername: true,
    hideTotalProfit: false,
    hideTotalWager: false,
  },
};

export const Default = () => <UserStatistics userStatistic={data.userInfo} />;
