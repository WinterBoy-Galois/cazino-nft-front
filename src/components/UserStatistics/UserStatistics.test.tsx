import React from 'react';
import { render } from '@testing-library/react';
import UserStatistics from '.';
import { GameTypes } from '../../models/gameTypes.model';

const userInfo = {
  id: '1',
  totalWager: 0,
  totalProfit: 0,
  mostPlayed: GameTypes.CLAMS,
  totalBets: 27,
  luckyBets: 4,
};

describe('UserStatistics', () => {
  it('should match snapshot', () => {
    // Arrange

    // Act
    const container = render(<UserStatistics userStatistic={userInfo} />);

    // Assert
    expect(container).toMatchSnapshot();
  });
});
