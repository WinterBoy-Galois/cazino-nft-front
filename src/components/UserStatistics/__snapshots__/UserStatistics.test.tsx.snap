// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserStatistics should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ul
        class="list"
      >
        <li
          class="list__item"
        >
          <div
            class="list__item__label truncate"
          >
            Total Wager
          </div>
          <div
            class="list__item__value"
          >
            <div
              class="container value"
              data-tag="allowRowEvents"
            >
              <svg
                class="icon"
                viewBox="0 0 64 64"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  transform="translate(0.00630876,-0.00301984)"
                >
                  <path
                    d="m63.033,39.744c-4.274,17.143-21.637,27.576-38.782,23.301-17.138-4.274-27.571-21.638-23.295-38.78,4.272-17.145,21.635-27.579,38.775-23.305,17.144,4.274,27.576,21.64,23.302,38.784z"
                  />
                  <path
                    class="icon__inner"
                    d="m46.103,27.444c0.637-4.258-2.605-6.547-7.038-8.074l1.438-5.768-3.511-0.875-1.4,5.616c-0.923-0.23-1.871-0.447-2.813-0.662l1.41-5.653-3.509-0.875-1.439,5.766c-0.764-0.174-1.514-0.346-2.242-0.527l0.004-0.018-4.842-1.209-0.934,3.75s2.605,0.597,2.55,0.634c1.422,0.355,1.679,1.296,1.636,2.042l-1.638,6.571c0.098,0.025,0.225,0.061,0.365,0.117-0.117-0.029-0.242-0.061-0.371-0.092l-2.296,9.205c-0.174,0.432-0.615,1.08-1.609,0.834,0.035,0.051-2.552-0.637-2.552-0.637l-1.743,4.019,4.569,1.139c0.85,0.213,1.683,0.436,2.503,0.646l-1.453,5.834,3.507,0.875,1.439-5.772c0.958,0.26,1.888,0.5,2.798,0.726l-1.434,5.745,3.511,0.875,1.453-5.823c5.987,1.133,10.489,0.676,12.384-4.739,1.527-4.36-0.076-6.875-3.226-8.515,2.294-0.529,4.022-2.038,4.483-5.155zm-8.022,11.249c-1.085,4.36-8.426,2.003-10.806,1.412l1.928-7.729c2.38,0.594,10.012,1.77,8.878,6.317zm1.086-11.312c-0.99,3.966-7.1,1.951-9.082,1.457l1.748-7.01c1.982,0.494,8.365,1.416,7.334,5.553z"
                  />
                </g>
              </svg>
              <span
                class="truncate"
                data-tag="allowRowEvents"
              >
                0.00000000
              </span>
            </div>
          </div>
        </li>
        <li
          class="list__item"
        >
          <div
            class="list__item__label truncate"
          >
            Total Profit
          </div>
          <div
            class="list__item__value"
          >
            <div
              class="container  value"
              data-tag="allowRowEvents"
            >
              <svg
                class="icon"
                viewBox="0 0 64 64"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  transform="translate(0.00630876,-0.00301984)"
                >
                  <path
                    d="m63.033,39.744c-4.274,17.143-21.637,27.576-38.782,23.301-17.138-4.274-27.571-21.638-23.295-38.78,4.272-17.145,21.635-27.579,38.775-23.305,17.144,4.274,27.576,21.64,23.302,38.784z"
                  />
                  <path
                    class="icon__inner"
                    d="m46.103,27.444c0.637-4.258-2.605-6.547-7.038-8.074l1.438-5.768-3.511-0.875-1.4,5.616c-0.923-0.23-1.871-0.447-2.813-0.662l1.41-5.653-3.509-0.875-1.439,5.766c-0.764-0.174-1.514-0.346-2.242-0.527l0.004-0.018-4.842-1.209-0.934,3.75s2.605,0.597,2.55,0.634c1.422,0.355,1.679,1.296,1.636,2.042l-1.638,6.571c0.098,0.025,0.225,0.061,0.365,0.117-0.117-0.029-0.242-0.061-0.371-0.092l-2.296,9.205c-0.174,0.432-0.615,1.08-1.609,0.834,0.035,0.051-2.552-0.637-2.552-0.637l-1.743,4.019,4.569,1.139c0.85,0.213,1.683,0.436,2.503,0.646l-1.453,5.834,3.507,0.875,1.439-5.772c0.958,0.26,1.888,0.5,2.798,0.726l-1.434,5.745,3.511,0.875,1.453-5.823c5.987,1.133,10.489,0.676,12.384-4.739,1.527-4.36-0.076-6.875-3.226-8.515,2.294-0.529,4.022-2.038,4.483-5.155zm-8.022,11.249c-1.085,4.36-8.426,2.003-10.806,1.412l1.928-7.729c2.38,0.594,10.012,1.77,8.878,6.317zm1.086-11.312c-0.99,3.966-7.1,1.951-9.082,1.457l1.748-7.01c1.982,0.494,8.365,1.416,7.334,5.553z"
                  />
                </g>
              </svg>
              <span
                class="truncate"
                data-tag="allowRowEvents"
              >
                0.00000000
              </span>
            </div>
          </div>
        </li>
        <li
          class="list__item"
        >
          <div
            class="list__item__label truncate"
          >
            Most Played
          </div>
          <div
            class="list__item__value"
          >
            <svg
              class="icon"
              viewBox="0 0 35 35"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                data-name="Layer 2"
                id="Layer_2"
              >
                <g
                  data-name="Layer 2"
                  id="Layer_2-2"
                >
                  <circle
                    cx="17.5"
                    cy="17.5"
                    fill="#fff"
                    r="17.5"
                  />
                  <path
                    class="cls-2"
                    d="M17.32 21.64zM33.37 17.73a.41.41 0 000-.11 1.24 1.24 0 00-.09-.38 1.17 1.17 0 00-.31-.46l-1.16-1a19.05 19.05 0 00-2.35-4.61 2 2 0 00-1.78-.83c-.3 0-.6.06-.89.1a18.44 18.44 0 00-1.16-1.83 2.78 2.78 0 00-2.46-1.14c-.73.06-1.44.16-2.1.28a4.53 4.53 0 00-3.45-1.63h-.54a4.54 4.54 0 00-3.42 1.6c-.62-.1-1.29-.19-2-.25a2.78 2.78 0 00-2.43 1.11 16.76 16.76 0 00-1.16 1.83c-.29 0-.59-.08-.89-.1a2 2 0 00-1.79.83 19.51 19.51 0 00-2.34 4.61l-1.16 1a1.2 1.2 0 00-.32.46 1.24 1.24 0 00-.09.38.41.41 0 000 .11.93.93 0 000 .33v.12a1.23 1.23 0 00.21.39 1.15 1.15 0 00.91.45 1.18 1.18 0 00.78-.3l2-1.77a.89.89 0 01.92-.18l2.95 1.1a3.21 3.21 0 003.1-.48l4.47-3.48a.9.9 0 011.12 0l4.48 3.48a3.21 3.21 0 003.1.48l2.94-1.1a.91.91 0 01.49 0 .88.88 0 01.44.22l2 1.77a1.15 1.15 0 001.69-.15 1.2 1.2 0 00.2-.39.56.56 0 000-.12 1.84 1.84 0 00.09-.34zM30.51 22.12v-.08a1.32 1.32 0 00-.1-.39 1.22 1.22 0 00-.67-.65l-1.61-.6a4.05 4.05 0 00-1-.23 3.78 3.78 0 00-2.26.46l-1.32.73a1.29 1.29 0 01-.78.17 1.44 1.44 0 01-.54-.18l-2.86-1.68a4.1 4.1 0 00-1.86-.56h-.36a4.1 4.1 0 00-1.86.56l-2.86 1.68a1.44 1.44 0 01-.54.18 1.31 1.31 0 01-.79-.17l-1.78-1a1.8 1.8 0 00-.63-.21 4.59 4.59 0 00-2.14.23l-1.6.6a1.21 1.21 0 00-.68.65 1.47 1.47 0 00-.1.4v.07a1.41 1.41 0 00.07.5 1.18 1.18 0 001.54.76L7 22.89a8.18 8.18 0 001 1.28 1.78 1.78 0 001.37.47c.23 0 .46 0 .68-.06a9.29 9.29 0 00.9 1 2.48 2.48 0 001.88.65c.52 0 1-.09 1.51-.15l.07.06a4.12 4.12 0 002.56.84h.42a4.36 4.36 0 002.21-.59 3.7 3.7 0 00.43-.32c.51.07 1.06.12 1.62.16a2.5 2.5 0 001.89-.65 10.57 10.57 0 00.89-1c.22 0 .45.05.68.06a1.78 1.78 0 001.37-.47 8.18 8.18 0 001-1.28l1.3.49a1.18 1.18 0 001.54-.76 1.21 1.21 0 00.19-.5z"
                  />
                </g>
              </g>
            </svg>
            CLAMS
          </div>
        </li>
        <li
          class="list__item"
        >
          <div
            class="list__item__label truncate"
          >
            Total Bets
          </div>
          <div
            class="list__item__value"
          >
            27
          </div>
        </li>
        <li
          class="list__item"
        >
          <div
            class="list__item__label truncate"
          >
            Won Bets
          </div>
          <div
            class="list__item__value"
          >
            4
          </div>
        </li>
        <li
          class="list__item"
        >
          <div
            class="list__item__label truncate"
          >
            Lost Bets
          </div>
          <div
            class="list__item__value"
          >
            23
          </div>
        </li>
      </ul>
    </div>
  </body>,
  "container": <div>
    <ul
      class="list"
    >
      <li
        class="list__item"
      >
        <div
          class="list__item__label truncate"
        >
          Total Wager
        </div>
        <div
          class="list__item__value"
        >
          <div
            class="container value"
            data-tag="allowRowEvents"
          >
            <svg
              class="icon"
              viewBox="0 0 64 64"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                transform="translate(0.00630876,-0.00301984)"
              >
                <path
                  d="m63.033,39.744c-4.274,17.143-21.637,27.576-38.782,23.301-17.138-4.274-27.571-21.638-23.295-38.78,4.272-17.145,21.635-27.579,38.775-23.305,17.144,4.274,27.576,21.64,23.302,38.784z"
                />
                <path
                  class="icon__inner"
                  d="m46.103,27.444c0.637-4.258-2.605-6.547-7.038-8.074l1.438-5.768-3.511-0.875-1.4,5.616c-0.923-0.23-1.871-0.447-2.813-0.662l1.41-5.653-3.509-0.875-1.439,5.766c-0.764-0.174-1.514-0.346-2.242-0.527l0.004-0.018-4.842-1.209-0.934,3.75s2.605,0.597,2.55,0.634c1.422,0.355,1.679,1.296,1.636,2.042l-1.638,6.571c0.098,0.025,0.225,0.061,0.365,0.117-0.117-0.029-0.242-0.061-0.371-0.092l-2.296,9.205c-0.174,0.432-0.615,1.08-1.609,0.834,0.035,0.051-2.552-0.637-2.552-0.637l-1.743,4.019,4.569,1.139c0.85,0.213,1.683,0.436,2.503,0.646l-1.453,5.834,3.507,0.875,1.439-5.772c0.958,0.26,1.888,0.5,2.798,0.726l-1.434,5.745,3.511,0.875,1.453-5.823c5.987,1.133,10.489,0.676,12.384-4.739,1.527-4.36-0.076-6.875-3.226-8.515,2.294-0.529,4.022-2.038,4.483-5.155zm-8.022,11.249c-1.085,4.36-8.426,2.003-10.806,1.412l1.928-7.729c2.38,0.594,10.012,1.77,8.878,6.317zm1.086-11.312c-0.99,3.966-7.1,1.951-9.082,1.457l1.748-7.01c1.982,0.494,8.365,1.416,7.334,5.553z"
                />
              </g>
            </svg>
            <span
              class="truncate"
              data-tag="allowRowEvents"
            >
              0.00000000
            </span>
          </div>
        </div>
      </li>
      <li
        class="list__item"
      >
        <div
          class="list__item__label truncate"
        >
          Total Profit
        </div>
        <div
          class="list__item__value"
        >
          <div
            class="container  value"
            data-tag="allowRowEvents"
          >
            <svg
              class="icon"
              viewBox="0 0 64 64"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                transform="translate(0.00630876,-0.00301984)"
              >
                <path
                  d="m63.033,39.744c-4.274,17.143-21.637,27.576-38.782,23.301-17.138-4.274-27.571-21.638-23.295-38.78,4.272-17.145,21.635-27.579,38.775-23.305,17.144,4.274,27.576,21.64,23.302,38.784z"
                />
                <path
                  class="icon__inner"
                  d="m46.103,27.444c0.637-4.258-2.605-6.547-7.038-8.074l1.438-5.768-3.511-0.875-1.4,5.616c-0.923-0.23-1.871-0.447-2.813-0.662l1.41-5.653-3.509-0.875-1.439,5.766c-0.764-0.174-1.514-0.346-2.242-0.527l0.004-0.018-4.842-1.209-0.934,3.75s2.605,0.597,2.55,0.634c1.422,0.355,1.679,1.296,1.636,2.042l-1.638,6.571c0.098,0.025,0.225,0.061,0.365,0.117-0.117-0.029-0.242-0.061-0.371-0.092l-2.296,9.205c-0.174,0.432-0.615,1.08-1.609,0.834,0.035,0.051-2.552-0.637-2.552-0.637l-1.743,4.019,4.569,1.139c0.85,0.213,1.683,0.436,2.503,0.646l-1.453,5.834,3.507,0.875,1.439-5.772c0.958,0.26,1.888,0.5,2.798,0.726l-1.434,5.745,3.511,0.875,1.453-5.823c5.987,1.133,10.489,0.676,12.384-4.739,1.527-4.36-0.076-6.875-3.226-8.515,2.294-0.529,4.022-2.038,4.483-5.155zm-8.022,11.249c-1.085,4.36-8.426,2.003-10.806,1.412l1.928-7.729c2.38,0.594,10.012,1.77,8.878,6.317zm1.086-11.312c-0.99,3.966-7.1,1.951-9.082,1.457l1.748-7.01c1.982,0.494,8.365,1.416,7.334,5.553z"
                />
              </g>
            </svg>
            <span
              class="truncate"
              data-tag="allowRowEvents"
            >
              0.00000000
            </span>
          </div>
        </div>
      </li>
      <li
        class="list__item"
      >
        <div
          class="list__item__label truncate"
        >
          Most Played
        </div>
        <div
          class="list__item__value"
        >
          <svg
            class="icon"
            viewBox="0 0 35 35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              data-name="Layer 2"
              id="Layer_2"
            >
              <g
                data-name="Layer 2"
                id="Layer_2-2"
              >
                <circle
                  cx="17.5"
                  cy="17.5"
                  fill="#fff"
                  r="17.5"
                />
                <path
                  class="cls-2"
                  d="M17.32 21.64zM33.37 17.73a.41.41 0 000-.11 1.24 1.24 0 00-.09-.38 1.17 1.17 0 00-.31-.46l-1.16-1a19.05 19.05 0 00-2.35-4.61 2 2 0 00-1.78-.83c-.3 0-.6.06-.89.1a18.44 18.44 0 00-1.16-1.83 2.78 2.78 0 00-2.46-1.14c-.73.06-1.44.16-2.1.28a4.53 4.53 0 00-3.45-1.63h-.54a4.54 4.54 0 00-3.42 1.6c-.62-.1-1.29-.19-2-.25a2.78 2.78 0 00-2.43 1.11 16.76 16.76 0 00-1.16 1.83c-.29 0-.59-.08-.89-.1a2 2 0 00-1.79.83 19.51 19.51 0 00-2.34 4.61l-1.16 1a1.2 1.2 0 00-.32.46 1.24 1.24 0 00-.09.38.41.41 0 000 .11.93.93 0 000 .33v.12a1.23 1.23 0 00.21.39 1.15 1.15 0 00.91.45 1.18 1.18 0 00.78-.3l2-1.77a.89.89 0 01.92-.18l2.95 1.1a3.21 3.21 0 003.1-.48l4.47-3.48a.9.9 0 011.12 0l4.48 3.48a3.21 3.21 0 003.1.48l2.94-1.1a.91.91 0 01.49 0 .88.88 0 01.44.22l2 1.77a1.15 1.15 0 001.69-.15 1.2 1.2 0 00.2-.39.56.56 0 000-.12 1.84 1.84 0 00.09-.34zM30.51 22.12v-.08a1.32 1.32 0 00-.1-.39 1.22 1.22 0 00-.67-.65l-1.61-.6a4.05 4.05 0 00-1-.23 3.78 3.78 0 00-2.26.46l-1.32.73a1.29 1.29 0 01-.78.17 1.44 1.44 0 01-.54-.18l-2.86-1.68a4.1 4.1 0 00-1.86-.56h-.36a4.1 4.1 0 00-1.86.56l-2.86 1.68a1.44 1.44 0 01-.54.18 1.31 1.31 0 01-.79-.17l-1.78-1a1.8 1.8 0 00-.63-.21 4.59 4.59 0 00-2.14.23l-1.6.6a1.21 1.21 0 00-.68.65 1.47 1.47 0 00-.1.4v.07a1.41 1.41 0 00.07.5 1.18 1.18 0 001.54.76L7 22.89a8.18 8.18 0 001 1.28 1.78 1.78 0 001.37.47c.23 0 .46 0 .68-.06a9.29 9.29 0 00.9 1 2.48 2.48 0 001.88.65c.52 0 1-.09 1.51-.15l.07.06a4.12 4.12 0 002.56.84h.42a4.36 4.36 0 002.21-.59 3.7 3.7 0 00.43-.32c.51.07 1.06.12 1.62.16a2.5 2.5 0 001.89-.65 10.57 10.57 0 00.89-1c.22 0 .45.05.68.06a1.78 1.78 0 001.37-.47 8.18 8.18 0 001-1.28l1.3.49a1.18 1.18 0 001.54-.76 1.21 1.21 0 00.19-.5z"
                />
              </g>
            </g>
          </svg>
          CLAMS
        </div>
      </li>
      <li
        class="list__item"
      >
        <div
          class="list__item__label truncate"
        >
          Total Bets
        </div>
        <div
          class="list__item__value"
        >
          27
        </div>
      </li>
      <li
        class="list__item"
      >
        <div
          class="list__item__label truncate"
        >
          Won Bets
        </div>
        <div
          class="list__item__value"
        >
          4
        </div>
      </li>
      <li
        class="list__item"
      >
        <div
          class="list__item__label truncate"
        >
          Lost Bets
        </div>
        <div
          class="list__item__value"
        >
          23
        </div>
      </li>
    </ul>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
